src/ui/setup/dynamic/devtools.ts

```diff
@@ -62,8 +62,7 @@ declare global {
 }
 
 export default async (store: Store) => {
-  const url = new URL(window.location.href);
-  const dispatchUrl = url.searchParams.get("dispatch") || process.env.NEXT_PUBLIC_DISPATCH_URL;
+  const dispatchUrl = process.env.NEXT_PUBLIC_DISPATCH_URL;
```

Comment:
I think we need to keep the QS parameter support around for now unless @jazzdan has already resolved this. Either way, this PR probably isn't the right place to make the change unless Nextjs broke it.